.idea/.idea.Snackr/.idea/contentModel.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You might want to add the .idea folder to your gitignore file so that it doesn't get committed to git hub since it's just a folder that rider uses.